### DISCUSSION TITLE
Simulation of 5th generation district

### DISCUSSION THREAD

**Discussion by KFSchlager** [architecture-building-systems/CityEnergyAnalyst#3644] [Q&A] [answered]
Hi,
I'm trying to simulate a 5th generation district (about 8°C) is there a way to do that with cea?
Thank you very much in advance!

**Comment by MatNif** [comment, marked as answer]
Hi @KFSchlager ,

Thanks for launching this discussion.

The operating temperature of the thermal networks is currently hard-coded, and the optimization algorithm only considers one type of thermal energy demand, i.e. space heating or space cooling. We've thought about extending the functionality of CEA to multi-energy systems optimization in the past, where you could also set the operating temperature of your thermal network. The framework of the code is, therefore, set in a way that would allow for this development to be made. However, realistically, we will not have the resources to include this in CEA any time soon.

As an open-source software, we are always looking for community contributions. So, if you want to join our developer community and propose/implement 5th-generation district energy solutions, feel free to open a GitHub issue and let us know about your project. We would happily assist you in reviewing and testing your code.
